recipe_cudagl.Dockerfile

```diff
@@ -0,0 +1,133 @@
+# syntax=docker/dockerfile:1.4
+
+FROM scratch as scripts
+
+COPY --chmod=755 <<EOF /setup
```

Comment:
uhhh. I did not realize you could execute arbitrary code in a COPY. what does this do with the files it builds?